fleetshard/pkg/central/cloudprovider/awsclient/rds_test.go

```diff
@@ -29,10 +29,18 @@ func newTestRDS() (*RDS, error) {
 		return nil, fmt.Errorf("unable to create RDS client: %w", err)
 	}
 
+	cfg := &config.ManagedDB{
```

Comment:
nit: I'd reuse the version from `envDefault` 
```suggestion
	cfg := &config.ManagedDB{}
	if err := env.Parse(cfg); err != nil {
		return nil, fmt.Errorf("unable to parse ManagedDB config: %w", err)
	}
	// Override the configuration for tests to save resources.
	cfg.BackupRetentionPeriod = 1
	cfg.MinCapacityACU = 0.5
	cfg.MaxCapacityACU = 1
```